packing_product_change/__manifest__.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+{'name': 'Product Equivalences',
+ 'version': '10.0.1.0.0',
+ 'license': 'AGPL-3',
+ 'author': "Camptocamp, "
+           "Serv. Tecnol. Avanzados - Pedro M. Baeza, "
```

Comment:
As far as I know @pedrobaeza  didn't contribute to this module as it comes from http://bazaar.launchpad.net/~camptocamp/c2c-ecom-addons/7.0/view/head:/packing_product_change/__openerp__.py. This line to be removed.